rules/properties-alphabetical-order/isOrderCorrect.js

```diff
@@ -0,0 +1,50 @@
+import { isStandardSyntaxProperty } from '../../utils/isStandardSyntaxProperty.js';
+import { isCustomProperty } from '../../utils/isCustomProperty.js';
+import * as vendor from '../../utils/vendor.js';
+import { checkAlphabeticalOrder } from '../checkAlphabeticalOrder.js';
+
+export function isOrderCorrect(node) {
```

Comment:
This is almost exact copy of https://github.com/hudochenkov/stylelint-order/blob/master/rules/properties-alphabetical-order/checkNode.js

Could you find a way to make it DRY? Because it is really important to have this logic the same for any type of checks the rule does.

Reply:
That's true. I also thought about it 💯